docs/trace/advanced/stratified-sampling-example/README.md

```diff
@@ -0,0 +1,103 @@
+# Stratified Sampling: An Example
+
+This example shows how to achieve stratified sampling in OpenTelemetry.NET.
+
+## What is stratified sampling?
+
+Stratified sampling is a way to divide a population into mutually exclusive
+sub-populations or "strata". For example, the strata for a population of
+"queries" could be "user-initiated queries" and "programmatic queries". Each
+stratum is then sampled using a probabilistic sampling method. This ensures
+that all sub-populations are represented.
+
+## How does this example do stratified sampling?
+
+We achieve this by using a custom Sampler that internally holds two samplers.
+Based on the stratum, the appropriate sampler is invoked.
+
+One prerequisite for this is that the tag (e.g. queryType) used for the
+stratified sampling decision must be provided as part of activity creation.
+
+We use disproportionate stratified sampling here - i.e., the sample size of
```

Comment:
I like the adjective "unequal probability" to describe this kind of disproportionate sampling, for what it's worth.